### DISCUSSION TITLE
USBSerialdrivers e QNAP bloccato

### DISCUSSION THREAD

**Discussion by gmarongiu71** [indomus/forum#1682] [Hardware & Firmware]
Ho un problema con questo driver, provato sia versione 4.4.1 che 5.0.0 ed il mio QNAP TS-251A con FW version QTS 5 e  ITEAD Sonoff ZigBee 3.0 USB Dongle Plus.
Se riavvio il NAS capita spesso che il driver mi impalli il NAS bloccando sia l'avvio che il riavvio obbligandomi allo spegnimento forzato. 
Se disinstallo il driver funziona tutto e se lo reinstallo di nuovo no, finchè ad un certo punto, "per caso" tra virgolette perchè nulla è per caso, inizia a funzionare fino al riavvio successivo.
Premetto che il problema lo fia sia con dongle inserito che no.
Ci sono altri con il mio stesso NAS ed il mio stesso problema? Se altri hanno il mio NAS senza problemi che chiavetta Zigbee e  che versione di driver hanno?
Grazie

**Comment by snakuzzo** [comment]
Io ormai ho dismesso il QNAP (che era quellp che trovi nella nostra recensione e con cui abbiamo fatto le prove).
Posso dirti che non ho mai avuto problemi né con QTS4 né con QTS5. Stesso dicasi per usb serial drivers 3 e 5.

**Reply by gmarongiu71** [reply]
Che chiavetta utilizzavi? Il problema l'ho solo da quando utilizzo i driver, prima di montare la chiavetta non ho mai avuto problemi.
Tra l'altro, davi un ordine di avvio? Se si come facevi?
Grazie

**Reply by snakuzzo** [reply]
Usato chiavette cc253x collegate per giorni e giorni con z2m come container docker. Non davo alcun ordine di avvio (non so nemmeno di preciso che intendi).

**Reply by gmarongiu71** [reply]
Intendo ordine di avvio delle applicazioni, ho visto che il driver lo avvia sempre per ultimo, quando tutto il resto è già su.
Il tuo sistema girava su Docker in container station? Ora che sistema usi?
Grazie

**Reply by gmarongiu71** [reply]
Comunque il problema l'ho al riavvio, se il sistema è su funziona tutto bene finchè anche per lunghissimi periodi senza problemi non riavvio

**Reply by snakuzzo** [reply]
Usavo docker e docker-compose.facevo tutto da riga di comando. Ma per i test ho usato anche solo container station.
Il qnap non lo riavviavo praticamente mai...solo per aggiornare qts...e mai avuto problemi

**Reply by gmarongiu71** [reply]
Allora provo anch'io a non riavviare più... Lo faccio solo per pulizia settimanale.
Ora su quale sistema hai HA?

**Reply by snakuzzo** [reply]
Minipc herobox (recensito) con debian. Sempre docker + docker-compose

**Comment by gmarongiu71** [comment]
Grazie, ora è miracolosamente partito... Non lo riavvio più.
